src/utils/utils.ts

```diff
@@ -47,12 +117,35 @@ export const getNumItems = async (uuid: string) => {
 
     if (response.status === 200) {
       const data = await response.json();
-      const numItems = data.nyplAPI.response.numItems;
-      return numItems;
+      const responseData = data.nyplAPI.response;
+      // if (apiUrl.includes('api/v2/items/total')) {
+      //   console.log("responseData for api/v2/items/total", responseData)
+      // }
+      return responseData;
     } else {
       return 0;
     }
   } catch (error) {
     return 0;
   }
 };
+
+export const getItemDataFromImageID = async (imageID: string) => {
+  const apiUri = await getAPIUri("local_image_id", imageID);
+  const data = await apiCall(apiUri.apiUri);
+  return {
+    uuid: apiUri.uuid,
+    title: getTitleFromRepoAPIResponseData(data) || "",
+  };
+};
+
+function addCommas(number) {
+  // Return the formatted number
+  return Number(number).toLocaleString("en-US");
+}
+
+function getTitleFromRepoAPIResponseData(data) {
+  // if titleInfo is an array of objects, use the first object. if titleInfo is a hash, return the title.
+  const titleInfo = data.mods.titleInfo;
+  return Array.isArray(titleInfo) ? titleInfo[0].title.$ : titleInfo.title.$;
```

Comment:
Using the first object in the array is fine. We chatted about this and using the `"usage": "primary"`.